src/main/java/fr/insalyon/creatis/gasw/script/ScriptGenerator.java

```diff
@@ -58,7 +61,7 @@ public synchronized static ScriptGenerator getInstance() throws GaswException {
         return instance;
     }
 
-    private ScriptGenerator() throws GaswException {
+    protected ScriptGenerator() throws GaswException {
```

Comment:
no need to make it protected if MoeurLiteScriptGenerator does not extend it.

Reply:
chganged to public